test/dummy/data/atoms_showcase.yml

```diff
@@ -102,6 +102,23 @@ atoms:
     - title: Title
       url: url
 
+  Dummy::Atom::Cards::Logo:
+    - title: Item title lorem ipsum damus
+      cover: true
+      url: "#url"
+      _count: 16
+
+    - title: Item title lorem ipsum damus
+      cover: true
+      _count: 10
+
+    - cover: true
+      url: "#url"
+      _count: 4
+
+    - cover: true
+      _count: 3
+
```

Comment:
prosim vzdy pri kodovani atomu do showcase vsechny mozne pripady, at je to poradne otestovane; tady jsem doplnil